### PR TITLE
Update local module options

### PR DESCRIPTION
**Pull request by itdependsnetworks**
##### SUMMARY

Update local module options

Update local module options to include ansible.cfg file.
##### ISSUE TYPE
- Docs Pull Request

+label: docsite_pr

##### ISSUE TYPE

- Docs Pull Request

##### COMPONENT NAME

Docs

##### ADDITIONAL INFORMATION

```paste below

```

### CODE REVIEW

File: docs/docsite/rst/dev_guide/developing_locally.rst [hunk 1]
```diff
@@ -75,6 +75,7 @@ Ansible loads plugins automatically too, loading each type of plugin separately
 You can create or add a local plugin in any of these locations:
 
 * any directory added to the relevant ``ANSIBLE_plugin_type_PLUGINS`` environment variable (these variables, such as ``$ANSIBLE_INVENTORY_PLUGINS`` and ``$ANSIBLE_VARS_PLUGINS`` take colon-separated lists like ``$PATH``)
+* Adding or adjusting the ``library`` parameter to the ``ansible.cfg`` file
```
Comment (bcoca) [inline docs/docsite/rst/dev_guide/developing_locally.rst]: I would point to the actual config setting DEFAULT_MODULE_PATH, which then details the different ways of setting it, not just the ini key w/o section information.

**Review by bcoca — CHANGES_REQUESTED** [review]

File: docs/docsite/rst/dev_guide/developing_locally.rst [hunk 2]
```diff
@@ -72,14 +72,16 @@ Ansible loads plugins automatically too, loading each type of plugin separately
...
+To get your local plugins loaded automatically, create or add them in any of these locations:
 
 * any directory added to the relevant ``ANSIBLE_plugin_type_PLUGINS`` environment variable (these variables, such as ``$ANSIBLE_INVENTORY_PLUGINS`` and ``$ANSIBLE_VARS_PLUGINS`` take colon-separated lists like ``$PATH``)
 * the directory named for the correct ``plugin_type`` within ``~/.ansible/plugins/`` - for example, ``~/.ansible/plugins/callback``
 * the directory named for the correct ``plugin_type`` within ``/usr/share/ansible/plugins/`` - for example, ``/usr/share/ansible/plugins/action``
 
 Once your plugin file is in one of these locations, Ansible will load it and you can use it in a any local module, task, playbook, or role.
 
+Alternatively, you can point a ``DEFAULT_MODULE_PATH`` in your ``ansible.cfg`` file to any directory that contains your local plugins.
+
```
Comment (bcoca) [inline docs/docsite/rst/dev_guide/developing_locally.rst:82]: this is conflating the 'setting name' with the ini section/key, I would just point to the link to the config page that explains the different ways of setting it.

Comment (bcoca) [reply docs/docsite/rst/dev_guide/developing_locally.rst:82]: also that is only for modules, you might want to point out that each plugin type has it's own config setting

**Review by bcoca — APPROVED** [review]

**Comment by itdependsnetworks** [comment]
Thanks, I wasn’t following, and meant I circle back and ask

**Comment by acozine** [comment]
Thanks @itdependsnetworks for suggesting updates to the ansible documentation. I hope the updated version of your changes still addresses your use case.

**Pull request merged**